inspirehep/dojson/hep/rules/bd9xx.py

```diff
@@ -236,20 +236,34 @@ def references2marc(self, key, value):
     journal_artid = get_value(reference, 'publication_info.artid')
     pubnote = build_pubnote(journal_title, journal_volume, journal_pg_start,
                             journal_pg_end, journal_artid)
+
+    doi = get_value(reference, 'dois[0]')
+    a_list = ['doi:{}'.format(doi) if doi else doi]
```

Comment:
instead of these two lines, you can do this to send back all dois:
`a_list = ['doi:{}'.format(doi) for doi in get_value(reference, 'dois', [])]`